scripts/hud/cgaz.ts

```diff
@@ -638,7 +736,7 @@ class Cgaz {
 
 				// arrow
 				if (this.primeArrowEnable) {
-					if (getSizeSquared2D(wishDir) > 0) {
+					if (MomMath.magnitude2D(wishDir) > 0) {
```

Comment:
this is an extra `sqrt` call

Reply:
Oops!! Good spot. I thought it just find/replaced all of these, this is concerning. I'll go through and double check everything.